### PR TITLE
ENH: Pretty Breaks classifier

### PR DESCRIPTION
**Pull request by sjsrey**

### CODE REVIEW

**Comment by jGaboardi** [comment]
Is there an associated ticket with this fucntionality?

File: mapclassify/classifiers.py [hunk 1]
```diff
@@ -1323,6 +1360,41 @@ def update(self, y=None, inplace=False, **kwargs):
             return new
 
 
+class PrettyBreaks(MapClassifier):
+    def __init__(self, y, k=5):
+        """
+        Pretty breakpoints
+
+        Computes breaks that are equally spaced round values which
+        cover the range of values in `y`. The breaks are chose so that
```
Comment (jGaboardi) [inline mapclassify/classifiers.py:1369]: chosen

**Review by jGaboardi — APPROVED** [review]

File: mapclassify/_classify_API.py [hunk 2]
```diff
@@ -33,6 +34,7 @@
     "naturalbreaks": NaturalBreaks,
     "quantiles": Quantiles,
     "percentiles": Percentiles,
+    "pretty": PrettyBreaks,
```
Comment (martinfleis) [inline mapclassify/_classify_API.py:37]: This is the only one where key doesn't match lowercase class. Should this be `"prettybreaks": PrettyBreaks` or should the class be called `Pretty` only if we want to use `pretty` as a key?

**Review by martinfleis — COMMENTED** [review]
One note on API. 

Can you also include an example in the ReadMe as we have for other classifiers?

**Comment by martinfleis** [comment]
Thanks @sjsrey!

**Pull request merged**